app/Repositories/Three/PostRepository.php

```diff
@@ -134,7 +134,7 @@ public function update($post, $data)
 
         // If the quantity was updated, update the total quantity on the signup.
         if (isset($data['quantity'])) {
-            $signup = Signup::find($post->signup_id);
+            $signup = $post->signup;
```

Comment:
👀

Reply:
credit to @DFurnes , always leaving things cleaner than he found them!